packages/lab/src/text-box/TextBox.tsx

```diff
@@ -34,7 +35,7 @@ const normalizeTextColor = createRuleNormalizer<TextColorProp>({
 
   primary: Color.Dark500,
   secondary: Color.Dark300,
-
+  'light-gray': Color.Dark100,
```

Comment:
How about calling it disabled?

Reply:
It's about color, not type, I can rename it